src/EditorFeatures/Core/FindUsages/ClassifiedSpansAndHighlightSpanFactory.cs

```diff
@@ -44,6 +49,46 @@ internal static class ClassifiedSpansAndHighlightSpanFactory
                 documentSpan.Document, documentSpan.SourceSpan, cancellationToken).ConfigureAwait(false);
         }
 
+        public static ClassifiedTextElement BuildClassifiedTextElementForClassifiedSpans(IEnumerable<ClassifiedSpan> classifiedSpans, ITextSnapshot snapshot, CancellationToken cancellationToken)
+        {
+            Contract.Assert(classifiedSpans != null && classifiedSpans.Any());
+
+            // Convert spans to textruns
+            var runs = new List<ClassifiedTextRun>();
+            var lastSpanEnd = -1;
+            foreach (var span in classifiedSpans)
+            {
+                // Add whitespace
+                if (lastSpanEnd > 0 && span.TextSpan.Start > lastSpanEnd)
```

Comment:
if difference between lastSpanEnd and newStart is 1. doesn't it mean there is no gap between 2?

Reply:
(definition of TextSpan.End) int End => Start + Length;  so the End is exclusive. if the difference between lastSpanEnd and newStart is 1, then there is a gap. @heejaechang